src/svelte-asset.js

```diff
@@ -39,7 +41,8 @@ class SvelteAsset extends Asset {
       generate: 'dom',
       format: 'cjs',
```

Comment:
Perhaps we should also use `major_version >= 3 ? 'esm' : 'es';` ?

Reply:
If it works yeah - that is correct.